Header header
float32 yaw_rate_cmd
float32 x_vel_cmd
float32 y_vel_cmd

float32 yaw_angle

float32[] OF_meas
float32[] OF_SF

float64 vicon_position_x
float64 vicon_position_y
float64 vicon_position_z
float64 vicon_orientation_x
float64 vicon_orientation_y
float64 vicon_orientation_z
float64 vicon_orientation_w

float32 vicon_vel_x
float32 vicon_vel_y
float32 vicon_vel_z
float32 vicon_angular_z

float32 vicon_accel_x
float32 vicon_accel_y
float32 vicon_accel_z

int16 dswitch



